webpack.config.js

```diff
@@ -0,0 +1,39 @@
+var fs = require('fs');
+var path = require('path');
+var webpack = require('webpack');
+var banner = `/**
+ * @license
+ * Copyright (c) 2015 Google
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+`;
+
+module.exports = {
+  entry: {
+    'webvr-polyfill': './src/main.js',
+    'webvr-polyfill.min': './src/main.js',
+  },
+  output: {
+    path: path.join(__dirname, 'build'),
+    filename: '[name].js',
+  },
+  resolve: {
+    extensions: ['.js', '.js', '.json']
+  },
+  plugins: [
+    new webpack.optimize.UglifyJsPlugin({
+      include: /\.min\.js$/,
+    }),
+    new webpack.BannerPlugin({ banner: banner, raw: true }),
+  ],
+};
```

Comment:
can you add a section for `devServer`?

after testing locally, here's a `diff` of the changes I made to this fil:

```diff
diff --git a/webpack.config.js b/webpack.config.js
index 35e5c47..27d107e 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -30,10 +30,19 @@ module.exports = {
   resolve: {
     extensions: ['.js', '.js', '.json']
   },
+  devtool: 'source-map',
   plugins: [
     new webpack.optimize.UglifyJsPlugin({
       include: /\.min\.js$/,
     }),
     new webpack.BannerPlugin({ banner: banner, raw: true }),
   ],
+  devServer: {
+    publicPath: '/build',
+    contentBase: [
+      path.resolve(__dirname, 'build'),
+      path.resolve(__dirname, 'examples')
+    ],
+    host: '0.0.0.0',
+  },
 };
```